sdk/src/Core/Amazon.Runtime/ClientConfig.cs

```diff
@@ -924,7 +923,7 @@ public int HttpClientCacheSize
                     return _httpClientCacheSize.Value;
                 }
 
-                return RuntimeInformation.IsOSPlatform(OSPlatform.Windows) ? 1 : Environment.ProcessorCount;
```

Comment:
Just to be safe can we add a preprocessor directive like 
`#IF NETCOREAPP3_1_OR_GREATER`

Reply:
How does that help given that you are not multi targeting your builds and NETCOREAPP3.1 is out of support?

Reply:
Even though it's out of support, there are still customers using older versions of netcore, so it's just a precaution

Reply:
In case my guess I was right I pushed an update. Otherwise just let me know and I will do another small change

Reply:
Yes that was what I was asking for thank you. Once it gets 2 approvers we can merge it in.